<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormOGG</class>
 <widget class="QWidget" name="FormOGG">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>OGG music information</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff/>
   </iconset>
  </property>
  <widget class="QLabel" name="bitRateL">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>130</y>
     <width>57</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>BR</string>
   </property>
  </widget>
  <widget class="QLabel" name="aBitRateL">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>110</y>
     <width>57</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>BR</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>140</y>
     <width>81</width>
     <height>27</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Bitrate:</string>
   </property>
  </widget>
  <widget class="QLabel" name="txtL">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>txt</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>51</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Path:</string>
   </property>
  </widget>
  <widget class="QLabel" name="chnL">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>110</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>mono/stereo</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pthE">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>381</width>
     <height>25</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="srateL">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>130</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>rate</string>
   </property>
  </widget>
  <widget class="QPushButton" name="tagB">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>110</y>
     <width>81</width>
     <height>28</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Edit tag</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>pthE</tabstop>
  <tabstop>tagB</tabstop>
  <tabstop>closeButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
